pages/agent/v3/aws.md.erb

```diff
@@ -7,11 +7,15 @@ The Buildkite Agent can be run on AWS using our Elastic CI Stack for AWS.
 ## Install the Agent on your own AWS instance
 
 To run the agent on your own AWS instances use whichever installer matches your
-instance type. For Amazon Linux use the [Red Hat/CentOS installer](/docs/agent/v3/redhat).
+instance type. For Amazon Linux 2 use the
+[Red Hat/CentOS installer](/docs/agent/v3/redhat).
 
-## Use our pre-built Elastic CI Stack for AWS
+## Use our Elastic CI Stack for AWS
 
-The [Elastic CI Stack for AWS](/docs/agent/v3/elastic_ci_aws) is a pre-built CloudFormation template
-that gives you an autoscaling Buildkite Agent cluster in your own AWS. The
-stack includes Docker, S3 and CloudWatch integration, and is suitable for
-parallelizing large legacy test suites, testing any Linux-based project that can run within Docker, and performing AWS ops related tasks.
+The [Elastic CI Stack for AWS](/docs/agent/v3/elastic_ci_aws) is a
+CloudFormation template to deploy an autoscaling Buildkite Agent cluster. The
+agent instances include Docker, S3 and CloudWatch integration.
+
+You can an Elastic CI Stack deployment to test Linux or Windows projects,
+parallelize large legacy test suites, run Docker containers or docker-compose
```

Comment:
Is `large legacy` an important modifier here, would it be better as just `large` whether they’re legacy or not?

Reply:
:100: